include/metal/list/at.hpp

```diff
@@ -87,10 +87,10 @@ namespace metal {
                 class _85, class _86, class _87, class _88, class _89,
                 class _90, class _91, class _92, class _93, class _94,
                 class _95, class _96, class _97, class _98, class _99,
+
                 class... tail>
```

Comment:
I propose these empty lines because they help to separate blocks of different template arguments. In my opinion, these empty lines significantly improve the readability. In some cases (see below) these empty lines are even more useful because they improve the alignment (readability) of the fast-tracking arguments.